akka-projection-core/src/main/scala/akka/projection/Projection.scala

```diff
@@ -57,11 +60,35 @@ trait Projection[Envelope] {
  */
 @InternalApi
 private[projection] object RunningProjection {
-  def withBackoff(source: Source[Done, _], settings: ProjectionSettings): Source[Done, _] =
+
+  def withBackoff(source: () => Source[Done, _], settings: ProjectionSettings): Source[Done, _] =
     RestartSource
       .onFailuresWithBackoff(settings.minBackoff, settings.maxBackoff, settings.randomFactor, settings.maxRestarts) {
-        () => source
+        () => source()
```

Comment:
👍 good catch

We needed to extend the tests to exercise the backoff. My bad, I should have added with on the original PR.